gradle.properties

```diff
@@ -1,2 +1,3 @@
 version=22.7.0-SNAPSHOT
 
+org.gradle.welcome=never
```

Comment:
How long until we can customize the gradle welcome build?

Reply:
it's once or never only for now. I just got tired of the gradle long welcome message in CI builds.